doc/reference/simple-scripts.md

```diff
@@ -128,11 +128,11 @@ In more detail, the four multisig constructors are:
 
 ## Time locking
 
-In the Allegra and later eras the simple multisig script language above is
-extended with two additional terms for expressing conditions on the time.
+In the Allegra and later eras, the simple multisig script language above is
+extended with two additional terms for expressing any time conditions.
```

Comment:
It's not any condition on time, only certain conditions can be expresses. For example I cannot express that it is a Tuesday.